frontend/rust-lib/flowy-server/src/af_cloud/server.rs

```diff
@@ -45,7 +45,12 @@ pub struct AppFlowyCloudServer {
 
 impl AppFlowyCloudServer {
   pub fn new(config: AFCloudConfiguration, enable_sync: bool, device_id: String) -> Self {
-    let api_client = AFCloudClient::new(&config.base_url, &config.ws_base_url, &config.gotrue_url);
+    let api_client = AFCloudClient::new(
+      &config.base_url,
+      &config.ws_base_url,
+      &config.gotrue_url,
+      ClientConfiguration::default(),
```

Comment:
You can merge the main branch first. The main branch of AppFlowy already resolve it.

Reply:
merged